package.json

```diff
@@ -375,6 +375,7 @@
     "jszip": "^3.8.0",
     "prettier-bytes": "^1.0.4",
     "qr-image": "^3.2.0",
+    "react-json-view": "^1.21.3",
```

Comment:
Curious this package is ok as`react-json-view` seems not updated for 2yr - https://www.npmjs.com/package/react-json-view.

Reply:
Changed to react-json-view-lite as it has no additional dependencies.